cpp/include/cudf/column/column.hpp

```diff
@@ -75,6 +76,21 @@ class column {
    */
   column(column&& other) noexcept;
 
+  /**
+   * @brief Move the contents from `other` to create a new column.
+   *
+   * After the move, `other.data() == NULL`
+   *
```

Comment:
We'd want to say `nullptr` instead of `NULL`, but I don't think we really want to specify this at all. In my understanding, the standard wouldn't guarantee anything about the state of a moved-from object except that it is "valid": http://eel.is/c++draft/lib.types.movedfrom

> moved-from objects shall be placed in a valid but unspecified state

```suggestion
```